trunk/examples/witness-checking/regression/correctness/gsv2008.i-witness.yaml

```diff
@@ -0,0 +1,27 @@
+-
+  entry_type: loop_invariant
+  metadata:
+    format_version: 0.1
+    uuid: "c294a826-5505-4952-9e7d-e00df27ec731"
+    creation_time: "2023-08-02T17:02:16+02:00"
+    producer:
+      name: UNUSED
+      version: UNUSED
+    task:
+      input_files:
+        - "C:\ultimate\trunk\examples\witness-checking\regression\correctness\gsv2008.i"
+      input_file_hashes:
+        "C:\ultimate\trunk\examples\witness-checking\regression\correctness\gsv2008.i": UNUSED
```

Comment:
here it is `UNUSED` instead of `hash` -- any particular reason? Do both strings have the same effect?